spec/features/indexing_integration_spec.rb

```diff
@@ -137,6 +137,30 @@
     end
   end
 
+  context 'an item with ALTO OCR' do
+    subject(:document) do
+      dor_harvester.document_builder.to_solr.first
+    end
+
+    let(:druid) { 'cc842mn9348' }
+
+    before do
+      fixture_file = File.new(File.join(FIXTURES_PATH, 'cc842mn9348_ocr_1.xml'))
+      stub_request(
+        :get,
+        'https://stacks.stanford.edu/file/cc842mn9348/EastTimor_CE-SPSC_Final_Decisions_2001_04b-2001_Sabino_Gouveia_Leite_Judgment_0001.xml'
+      ).to_return(body: fixture_file, status: 200)
+    end
+
+    it 'indexes all strings from the ALTO files into the full text field' do
+      fulltext = document[:full_text_tesimv]
+      expect(fulltext).to be_a Array
+      expect(fulltext.length).to eq 1
+      expect(fulltext.first).to start_with 'INTRODUCTION 1 The trial of'
+      expect(fulltext.first).to end_with 'rendering of the decision.'
```

Comment:
This test is now basically duplicated here and in the test for the class.  Not sure this one is necessary anymore, but kinda nice to know it works?  `/shrug`